tools/config/cmake.sh

```diff
@@ -6,12 +6,17 @@ CWD="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )" && cd "$CWD"
 ROOT_DIR=$(cd ../.. && pwd)
 [ -f "${ROOT_DIR}/res/cmake/def.cmake" ] || (echo "Corruption detected" && exit 1) # confirm correct directory
```

Comment:
maybe we need an error msg bit more explanatory: "Corruption detected: missing "def.cmake""

Reply:
more like "Code base in inconsisntent state - reacquire from source". ?

The reasoning for the specific `def.cmake` file just to be clear, is that the `def.cmake` is what checks if the `phare_configurator` is on, and if so, spawns the process calling this script. so if the def.cmake is not found, something is very wrong